### PR TITLE
Updated the tss lookup plugin to reflect breaking changes introduced in the underpinning SDK

### LINKED ISSUE #3057: community.general.tss Lookup Plugin throws exception when using python-tss-sdk 1.0.0.
### Summary

python-tss-sdk introduced a breaking change by removing the [SecretServerAccessError](https://github.com/thycotic/python-tss-sdk/blob/9727ffea263fd29ed7cb0a64325a95a735162365/thycotic/secrets/server.py#L106) class.

It appears to have been replaced with the [SecretServerClientError](https://github.com/thycotic/python-tss-sdk/blob/6969b8ee2161cab7c72ed01e1b7e6f1cdb2fd7f2/thycotic/secrets/server.py#L105) and [SecretServerServiceError](https://github.com/thycotic/python-tss-sdk/blob/6969b8ee2161cab7c72ed01e1b7e6f1cdb2fd7f2/thycotic/secrets/server.py#L109) classes.

An exception occurs while importing `SecretServerAccessError`

https://github.com/ansible-collections/community.general/blob/9631de49396ab0706df4a38c1cbab6a779a125dc/plugins/lookup/tss.py#L123

Resulting in the following AnsibleError being thrown.

https://github.com/ansible-collections/community.general/blob/9631de49396ab0706df4a38c1cbab6a779a125dc/plugins/lookup/tss.py#L143

### Issue Type

Bug Report

### Component Name

- plugins/lookup/tss.py

### Ansible Version

```console (paste below)
root@61dbe3fe35eb:/ansible# ansible --version
ansible 2.10.12
  config file = /ansible/ansible.cfg
  configured module search path = ['/root/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/local/lib/python3.8/dist-packages/ansible
  executable location = /usr/local/bin/ansible
  python version = 3.8.10 (default, Jun  2 2021, 10:49:15) [GCC 9.4.0]
```

### Configuration

```console (paste below)
root@61dbe3fe35eb:/ansible# ansible-config dump --only-changed
ANSIBLE_PIPELINING(/ansible/ansible.cfg) = True
DEFAULT_BECOME(/ansible/ansible.cfg) = True
DEFAULT_CALLBACK_PLUGIN_PATH(/ansible/ansible.cfg) = ['/etc/ansible/plugins/callback_plugins']
DEFAULT_FORKS(/ansible/ansible.cfg) = 25
DEFAULT_INVENTORY_PLUGIN_PATH(/ansible/ansible.cfg) = ['/ansible/inventory_plugins']
DEFAULT_LOAD_CALLBACK_PLUGINS(/ansible/ansible.cfg) = True
DEFAULT_MANAGED_STR(/ansible/ansible.cfg) = Ansible managed. Do NOT edit manually.
DEFAULT_ROLES_PATH(/ansible/ansible.cfg) = ['/ansible/roles']
HOST_KEY_CHECKING(/ansible/ansible.cfg) = False
HOST_PATTERN_MISMATCH(/ansible/ansible.cfg) = ignore
INVENTORY_ENABLED(/ansible/ansible.cfg) = ['pl_infoblox', 'pl_infrastructure', 'auto', 'yaml']
INVENTORY_UNPARSED_IS_FAILED(/ansible/ansible.cfg) = True
RETRY_FILES_ENABLED(/ansible/ansible.cfg) = False
```

### OS / Environment

Ubuntu 20.04

### Steps to Reproduce

requirements.txt
```text
ansible == 2.10.0
python-tss-sdk == 1.0.0
```

requirements.yml
```yaml
collections:
  - name: community.general
    version: 2.1.1
```

playbook.yml
```yaml
- hosts: localhost
  connection: local
  vars:
    thycotic_host: example.secretservercloud.com
    thycotic_username: "{{ tss_user }}"
    thycotic_password: "{{ tss_password }}"
    thycotic_base_uri: https://{{ thycotic_host }}
    secret_id: 1655
    secret: "{{ lookup('community.general.tss',
                                               registration_authority_secret_id,
                                               base_url=thycotic_base_uri,
                                               username=thycotic_username,
                                               password=thycotic_password)
                               }}"
     secret_dict: "{{ registration_authority_secret['items']
                                        | items2dict(key_name='slug', value_name='itemValue')
                                    }}"
    secret_value: "{{ secret_dict['password'] }}" 
  tasks:
  - name: display secret value
    ansible.builtin.debug:
      var: secret_value
```

command
```console
pip install -r requirments.txt
ansible-galaxy install -r requirements.yml
ansible-playbook -i localhost, playbook.yml
```

### Expected Results

I expect the password value to be displayed in an ansible debug statement.

### Actual Results

I get the following exception
```console (paste below)
Error was a <class 'ansible.errors.AnsibleError'>, original message: An unhandled exception occurred while running the lookup plugin 'community.general.tss'. Error was a <class 'ansible.errors.AnsibleError'>, original message: python-tss-sdk must be installed to use this plugin
```

### Code of Conduct

- [X] I agree to follow the Ansible Code of Conduct

### PR DESCRIPTION
**Pull request by EndlessTrax**
##### SUMMARY
This PR updates the plugin to bring it in line with the current `python-tss-sdk` version, which added breaking changes in v1.0.0. The plugin is now compatible with the latest SDK version.  

This also fixes #3057. 

##### ISSUE TYPE

- Bugfix Pull Request

##### COMPONENT NAME
`tss` lookup plugin

##### ADDITIONAL INFORMATION
No changes are needed to current playbooks, as inputs have stayed the same. Users will be encouraged to update their `python-tss-sdk` version to >1.0.0.

### CODE REVIEW

**Comment by amigus** [comment]
Ricky's changes should fix the problem so this PR is GTG from our perspective. Please merge if you are good with it.

File: plugins/lookup/tss.py [hunk 1]
```diff
@@ -136,7 +136,16 @@
 class LookupModule(LookupBase):
     @staticmethod
     def Client(server_parameters):
-        return SecretServer(**server_parameters)
+        authorizer = PasswordGrantAuthorizer(
+            server_parameters["base_url"],
+            server_parameters["username"],
+            server_parameters["password"],
+            server_parameters["token_path_uri"],
+        )
```
Comment (Ajpantuso) [inline plugins/lookup/tss.py:144]:
```suggestion
        if LooseVersion(thycotic_version) < LooseVersion('1.0.0'):
            return SecretServer(**server_parameters)
            
        authorizer = PasswordGrantAuthorizer(
            server_parameters["base_url"],
            server_parameters["username"],
            server_parameters["password"],
            server_parameters["token_path_uri"],
        )
```

`token_path_uri` is not a valid argument in `0.0.6` so preserving the existing behavior for versions < `1.0.0` should give appropriate time for deprecation (thycotic_version = `thycotic.__version__`). See [here](https://github.com/thycotic/python-tss-sdk/blob/9727ffea263fd29ed7cb0a64325a95a735162365/thycotic/secrets/server.py#L187-L193)

**Review by Ajpantuso — CHANGES_REQUESTED** [review]
Besides the comment on compatibility where will users be encouraged to upgrade to version `1.0.0`?

**Comment by felixfontein** [comment]
You also need a [changelog fragment](https://docs.ansible.com/ansible/devel/community/development_process.html#changelogs-how-to).

**Comment by EndlessTrax** [comment]
Thanks for the feedback @felixfontein. I've made updates as suggested. Let me know if there is anything else needed before merging. Cheers. 

cc: @amigus

File: changelogs/fragments/3139-tss-lookup-plugin-update-to-make-compatible-with-sdk-v1.yml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+bugfixes:
+  - tss_plugin - Fixed issue `#3057` by bringing the plugin in line with recent breaking changes the SecretServer Python SDK.
+    (https://github.com/ansible-collections/community.general/pull/3139)
```
Comment (Ajpantuso) [inline changelogs/fragments/3139-tss-lookup-plugin-update-to-make-compatible-with-sdk-v1.yml:3]:
```suggestion
  - tss lookup plugin - fixed incompatibility with ``python-tss-sdk`` version 1.0.0
    (https://github.com/ansible-collections/community.general/issues/3057, https://github.com/ansible-collections/community.general/pull/3139).
```

**Review by Ajpantuso — CHANGES_REQUESTED** [review]
Other than the changelog fragment this looks good.

**Comment by felixfontein** [comment]
@EndlessTrax thanks for fixing this!
@amigus @Ajpantuso thanks for reviewing!

**Review by Ajpantuso — APPROVED** [review]
LGTM, thanks!

**Pull request merged**